torch/csrc/jit/codegen/cuda/partition.cpp

```diff
@@ -104,9 +104,10 @@ bool hasReductionOperation(const Node* node) {
 bool maybeBroadcastOnShape(
     const Node* n,
     const std::vector<c10::optional<int64_t>>& shape) {
-  TORCH_INTERNAL_ASSERT(
-      n->outputs().size() == 1,
-      "not expecting multiple outputs from a node, graph partitioning logic needs to be updated");
+  // TORCH_INTERNAL_ASSERT(
```

Comment:
why is this commented out?

Reply:
`torch.native_layer_norm` and `torch.native_layer_norm_backward` generate 3 outputs apiece. Hence, this assertion will fail.